pkg/controller/flink/job_manager_controller.go

```diff
@@ -307,7 +307,7 @@ func jobmanagerTemplate(app *v1beta1.FlinkApplication) *v1.Deployment {
 		ObjectMeta: metaV1.ObjectMeta{
 			Namespace:   app.Namespace,
 			Labels:      labels,
-			Annotations: getCommonAnnotations(app),
+			Annotations: GetPrometheusAnnotations(config.GetConfig().PrometheusPort.Port),
```

Comment:
Probably out of the scope of this particular PR, but considering that other (future) annotations could also be added to the JM and TM, it may be worthwhile considering creating a function that can render annotations from different sources (with the Prometheus annotations being one of them)